src/engine/source/kvdb2/include/kvdb2/iKVDBManager.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef _IKVDBMANAGER_H
```

Comment:
I think it should be `_I_KVDB_MANAGER_H`

Reply:
Will be fixed in later documentation step.